src/app/admin-panel/admin-panel/students/students.component.html

```diff
@@ -1,18 +1,18 @@
 <p-fieldset legend="Активні класи" [toggleable]="true" [collapsed]="false">
   <span *ngFor="let  class_ of classes">
-    <button *ngIf="class_.isActive == true" type="button" pButton icon="pi pi-folder" label="&nbsp;&nbsp;{{class_.className}}" title="{{class_.id}}" (click)="loadStudents(class_.id)" class="ui-button-raised classButton"></button>
+    <button *ngIf="class_.isActive == true" type="button" pButton icon="pi pi-folder" label="&nbsp;&nbsp;{{class_.className}}" title="{{class_.id}}" (click)="loadStudents(class_.id); selectedClassNameHendler(class_.className + ' класу')" class="ui-button-raised classButton"></button>
```

Comment:
JFYI no need to compare boolean properties in conditions

Reply:
Changed.